pkg/discovery/dtofactory/cluster_dto_builder.go

```diff
@@ -195,9 +215,12 @@ func (builder *clusterDTOBuilder) createClusterData(clusterName string, namespac
 		}
 	}
 
+	// Post 8.2.3 kubeturbo sends all vcpu related commodities in millicores
```

Comment:
Nit: Maybe it is more clear to say: Starting from `8.2.5`, `kubeturbo` sends all vcpu related commodities in millicores

Reply:
Updated!